sensors/ev3_uart_sensor_defs.c

```diff
@@ -35,7 +35,7 @@ const struct ev3_uart_sensor_info ev3_uart_sensor_defs[] = {
 		.mode_info	= {
 			[0] = {
 				/**
-				 * @description: Reflected light<br />Red LED on
+				 * @description: Reflected light<br />LED color: red
```

Comment:
I was trying to make it consistent, just forgot to change this one. I'll fix it.